src/internal/m365/collection/exchange/transform.go

```diff
@@ -58,6 +58,28 @@ func toMessage(orig models.Messageable) models.Messageable {
 	return CloneMessageableFields(orig, message)
 }
 
+// eventAdditionalUnneededFields contains a set of fields that don't have API
+// support right now but are ignored by or may cause errors in M365 when
+// restoring items.
+var eventAdditionalUnneededFields = []string{
+	// Will cause errors about needing to put extension data in their own requests
+	// if not removed.
+	"iCalUId_v2",
+	// Appears to be duplicate of the iCalUId.
+	"uid",
+	// Navigation links from the calendar to the event itself.
+	"calendar@odata.associationLink",
+	"calendar@odata.navigationLink",
```

Comment:
Curious, does graph auto populate any of these fields after we restore? i.e.if we fetch the event after restoring it, do these fields appear empty or filled? Wondering if removing `navigationLink` and such has any UX impact to the mailbox owner.

Reply:
yes, it auto populates them. The field itself contains the calendar ID and has the form
```
https://graph.microsoft.com/beta/users('<user_name>')/calendars('<calendar_id>')/$ref
```